apps/andi/lib/andi/event/event_handler.ex

```diff
@@ -30,16 +30,22 @@ defmodule Andi.Event.EventHandler do
   @instance_name Andi.instance_name()
 
   def handle_event(%Brook.Event{type: dataset_update(), data: %Dataset{} = data, author: author}) do
-    dataset_update()
-    |> add_event_count(author, data.id)
+    case Organizations.get(data.organization_id) do
+      nil ->
+        Logger.error("Organization #{data.organization_id} does not exist. Discarding update event for dataset #{data.id}.")
```

Comment:
Would this error get propagated to the user? (@LtChae)

Reply:
No, the user has no visibility into issues on the event stream